bindings/dotnet/DotOpenDAL/DotOpenDAL.csproj

```diff
@@ -18,7 +18,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <None Include="../target/native/osx-aarch_64/libopendal_dotnet.dylib">
+        <None Include="../target/debug/libopendal_dotnet.so">
```

Comment:
We need to set up arch detection here.

Reply:
Seems not a blocker from this PR to get merged. Would you like to raise an issue instead?